### PR TITLE
Tweaks

### PR DESCRIPTION
**Pull request by longhotsummer**
* buttons to markup italics and bold
* center toasts, remove after 3 seconds
* smarter, problem-oriented cheatsheet
* show subsidiary works on work detail page

### CODE REVIEW

**Comment by longhotsummer** [comment]
Broken terms @ 2019-05-03 2019-06-25 09-32-32

Broken terms @ 2019-05-03 2019-06-25 09-33-23

Tables tester – Indigo Platform 2019-06-25 09-55-02

**Comment by longhotsummer** [comment]
TODO: spelling of ceteris paribus

File: indigo_app/templates/document/_cheatsheet.html [hunk 1]
```diff
@@ -2,100 +2,515 @@
...
+          <div class="card">
+            <a href="#cheat-italics" data-toggle="collapse" class="card-header">Make text italics</a>
+            <div id="cheat-italics" class="collapse" data-parent="#cheat-accordion">
+              <div class="card-body">
+                <p>Display a word or a sentence as italics using <code>//word//</code>. Do this for Latin, foreign and other special terms.</p>
+                <p>For editorial remarks, use <code>[[remark]]</code> instead.</p>
+
+                <div class="row">
+                  <div class="col">
+                    <pre class="editor-snippet">The situation is same, //ceteris parabis//.</pre>
```
Comment (goose-life) [inline indigo_app/templates/document/_cheatsheet.html]:
```suggestion
                    <pre class="editor-snippet">The situation is same, //ceteris paribus//.</pre>
```

File: indigo_app/templates/document/_cheatsheet.html [hunk 2]
```diff
@@ -2,100 +2,515 @@
...
+                  </div>
+                  <div class="col">
+                    <div class="akoma-ntoso akoma-ntoso-snippet">
+                      <span class="akn-p mb-0">The situation is the same, <i>ceteris parabis</i>.</span>
```
Comment (goose-life) [inline indigo_app/templates/document/_cheatsheet.html]:
```suggestion
                      <span class="akn-p mb-0">The situation is the same, <i>ceteris paribus</i>.</span>
```

File: indigo_app/templates/document/_cheatsheet.html [hunk 3]
```diff
@@ -2,100 +2,515 @@
   <div class="modal-dialog modal-lg cheatsheet" role="document">
     <div class="modal-content">
       <div class="modal-header">
-        <h5 class="modal-title">Editor Cheatsheet</h5>
+        <h5 class="modal-title">Editor Help</h5>
```
Comment (goose-life) [inline indigo_app/templates/document/_cheatsheet.html]:
```suggestion
        <h5 class="modal-title">Editor help</h5>
```

File: indigo_app/templates/document/_cheatsheet.html [hunk 4]
```diff
@@ -2,100 +2,515 @@
...
+
+        <div class="accordion" id="cheat-accordion">
+          {% block cards %}
+
+          {% block cheat-preface %}
+          <div class="card">
+            <a href="#cheat-preface" data-toggle="collapse" class="card-header">Starting the Preface</a>
+            <div id="cheat-preface" class="collapse" data-parent="#cheat-accordion">
+              <div class="card-body">
+                <p>Mark the start of the Preface before the Preamble and body.</p>
```
Comment (goose-life) [inline indigo_app/templates/document/_cheatsheet.html]:
```suggestion
                <p>Mark the start of the Preface, before the Preamble and body.</p>
```

File: indigo_app/templates/document/_cheatsheet.html [hunk 5]
```diff
@@ -2,100 +2,515 @@
...
+            </div>
+          </div>
+          {% endblock %}
+
+          {% block cheat-longtitle %}
+          <div class="card">
+            <a href="#cheat-longtitle" data-toggle="collapse" class="card-header">Markup the long title</a>
+            <div id="cheat-longtitle" class="collapse" data-parent="#cheat-accordion">
+              <div class="card-body">
+                <p>Indicate the long title of the work using <code>PREFACE</code>. This usually comes just before the sentence "BE IT ENACTED..."</p>
```
Comment (goose-life) [inline indigo_app/templates/document/_cheatsheet.html]:
```suggestion
                <p>Indicate the long title of the work using <code>LONGTITLE</code>. This usually comes just before the sentence "BE IT ENACTED..."</p>
```

File: indigo_app/templates/document/_cheatsheet.html [hunk 6]
```diff
@@ -2,100 +2,515 @@
...
+            </div>
+          </div>
+          {% endblock %}
+
+          {% block cheat-remark %}
+          <div class="card">
+            <a href="#cheat-remark" data-toggle="collapse" class="card-header">Add an editorial remark</a>
+            <div id="cheat-remark" class="collapse" data-parent="#cheat-accordion">
+              <div class="card-body">
+                <p>Add an editorial remark, that isn't a part of the original document, using <code>[[remark]]</code>.</p>
```
Comment (goose-life) [inline indigo_app/templates/document/_cheatsheet.html]:
```suggestion
                <p>Add an editorial remark, text that isn't part of the original document, using <code>[[remark]]</code>.</p>
```

File: indigo_app/templates/document/_cheatsheet.html [hunk 7]
```diff
@@ -2,100 +2,515 @@
...
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+          {% endblock %}
+
+          {% block cheat-escape %}
+          <div class="card">
+            <a href="#cheat-escape" data-toggle="collapse" class="card-header">Add text that looks like a chapter, part or section but isn't</a>
```
Comment (goose-life) [inline indigo_app/templates/document/_cheatsheet.html]:
```suggestion
            <a href="#cheat-escape" data-toggle="collapse" class="card-header">Ignore text that looks like a chapter, part or section but isn't</a>
```

**Pull request merged**